packages/angular/cli/lib/config/schema.json

```diff
@@ -875,6 +875,12 @@
               "description": "Enables conditionally loaded ES2015 polyfills.",
               "type": "boolean",
               "default": false
+            },
+            "rebaseRootRelativeCssUrls": {
+              "description": "Change root relative URLs in stylesheets to include base HREF and deploy URL. Use only for compatibility and transition.  This behavior of this option is non-standard and will be removed in the next major release.",
```

Comment:
Nit This behavior of this option  -> The behavior of this option

Also note you have a double space before ‘this’

Reply:
Done.